server/app.ts

```diff
@@ -0,0 +1,56 @@
+import * as fs from "fs";
+import * as path from "path";
+import * as express from "express";
+import * as compression from "compression";
+import * as bodyParser from "body-parser";
+
+import { graphqlExpress, graphiqlExpress } from 'graphql-server-express';
+import { makeExecutableSchema } from 'graphql-tools';
+
+import { PORT } from './common';
+import * as plugins from './plugins/api';
+import { GenericNotifier } from './plugins/api/GenericNotifier';
+import { APIReturn } from './plugins/api/APIReturn';
+
+const typeDefs = fs.readFileSync(path.resolve(__dirname, "../api.graphql"), "utf8");
+
+const app = express();
+
+app.use(compression());
+process.on("unhandledRejection", err => {
+	throw err;
+});
+
+interface IStatusReturn {
+	[medium: string]: APIReturn;
+}
+
+const resolvers = {
+	Query: {
+		send_message: (prev: any, args: any) => {
+			let statusRet: IStatusReturn = {};
+			const src = Object.keys(args.plugins);
+			src.forEach( name => {
+				const message = args.message;
+				const config = { ...args.plugins[name], message };
+				const plugin: GenericNotifier = plugins.mediaAPI[name.toUpperCase()];
+				statusRet[name] = plugin.sendMessage(config);
```

Comment:
This should be wrapped in a `try {} catch {}` and a failure should be recorded for `statusRet` if an exception occurs.